libgnucash/engine/gnc-commodity.h

```diff
@@ -109,12 +109,10 @@ GType gnc_commodity_namespace_get_type(void);
 /* The ISO define is deprecated in favor of CURRENCY */
 #define GNC_COMMODITY_NS_ISO    "ISO4217"
 #define GNC_COMMODITY_NS_CURRENCY "CURRENCY"
-#define GNC_COMMODITY_NS_NASDAQ "NASDAQ"
-#define GNC_COMMODITY_NS_NYSE   "NYSE"
-#define GNC_COMMODITY_NS_EUREX  "EUREX"
-#define GNC_COMMODITY_NS_MUTUAL "FUND"
-#define GNC_COMMODITY_NS_AMEX   "AMEX"
-#define GNC_COMMODITY_NS_ASX    "ASX"
+#define GNC_COMMODITY_NS_BOND N_("BOND")
+#define GNC_COMMODITY_NS_SHARE N_("SHARE")
+#define GNC_COMMODITY_NS_MUTUAL _("FUND")
+#define GNC_COMMODITY_NS_OPTION _("OPTION")
```

Comment:
The missng Ns was the culprit. No idea, why that change was incomplete.

Reply:
Shouldn't `"CURRENCY"` be marked for translation too?